examples/federatedStorage/azure/federated-store.yaml

```
@@ -0,0 +1,9 @@
+type: AZURE
+config:
+  storage_account: "kubecost-multi-cluster-storage"
```

```suggestion
  storage_account: kubecost-multi-cluster-storage
```

Minor detail. All your other federated-store.yaml configs don't have quotes here